ykfr/src/llvmbridge.rs

```diff
@@ -23,6 +24,7 @@ impl Module {
         let mut module: MaybeUninit<LLVMModuleRef> = MaybeUninit::uninit();
         LLVMParseBitcodeInContext2(context, membuf, module.as_mut_ptr());
```

Comment:
Can you add and link to an issue (if one doesn't already exist) saying that this should use the loading logic from ykllvmwrap to take advantage of a thread's cache of the IR (instead of loading it every time).